pkg/sentry/fs/proc/net.go

```diff
@@ -500,7 +543,7 @@ func (n *netUDP) ReadSeqFileData(ctx context.Context, h seqfile.SeqHandle) ([]se
 	}
 
 	var buf bytes.Buffer
-	for _, se := range n.k.ListSockets() {
+	for _, se := range k.ListSockets() {
```

Comment:
/b/f/w/pkg/sentry/fs/proc/net.go:546:21: undefined: k

Reply:
Sorry about the compile error. Was introduced in code rebase. I shall have checked that before sending it out.